src/fireedge/etc/sunstone/admin/user-tab.yaml

```diff
@@ -0,0 +1,86 @@
+# -------------------------------------------------------------------------- #
+# Copyright 2002-2022, OpenNebula Project, OpenNebula Systems                #
+#                                                                            #
+# Licensed under the Apache License, Version 2.0 (the "License"); you may    #
+# not use this file except in compliance with the License. You may obtain    #
+# a copy of the License at                                                   #
+#                                                                            #
+# http://www.apache.org/licenses/LICENSE-2.0                                 #
+#                                                                            #
+# Unless required by applicable law or agreed to in writing, software        #
+# distributed under the License is distributed on an "AS IS" BASIS,          #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
+# See the License for the specific language governing permissions and        #
+# limitations under the License.                                             #
+#--------------------------------------------------------------------------- #
+
+---
+# This file describes the information and actions available in the USER tab
+
+# Resource
+
+resource_name: "USER"
+
+# Actions - Which buttons are visible to operate over the resources
+
+actions:
+  create_dialog: true
+  chgrp: true
+  delete: true
+  enable: true
+  disable: true
+
+# Filters - List of criteria to filter the resources
+
+filters:
+  label: true
+  state: true
+
+# Info Tabs - Which info tabs are used to show extended information
+
+info-tabs:
+
+  info:
+    enabled: true
+    information_panel:
+      enabled: true
+      actions:
+        rename: true
+    attributes_panel:
+      enabled: true
+      actions:
+        copy: true
+        add: true
+        edit: true
+        delete: true
+
+  group:
+    enabled: true
+    actions:
+      chgrp: true
+
+  quota:
+    enabled: true
+    actions:
+      quotas_dialog: true
+
+  accounting:
+    enabled: true
+    actions:
+      get_accounting: true
+
+  showback:
+    enabled: true
+    actions:
+      get_showback: true
+
+  authentication:
+    enabled: true
+    actions:
+      change_authentication: true
+      update_password: true
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)